leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -343,11 +361,11 @@ var ElementInspector = Backbone.View.extend({
             }
         }
 
-        if (this.intention.dynamicFunction.hasRepeat()) {
-            var repBegin = this.intention.dynamicFunction.getStartRepeatEpoch();
-            var repEnd = this.intention.dynamicFunction.getEndRepeatEpoch();
-            var repNum = this.intention.dynamicFunction.getRepeatRepNum();
-            var absTime = this.intention.dynamicFunction.getRepeatAbsTime();
+        if (this.model.get('intention').get('evolvingFunction').get('hasRepeat')) {
+            var repBegin = this.model.get('intention').get('evolvingFunction').get('repStart');
+            var repEnd = this.model.get('intention').get('evolvingFunction').get('repStop');
+            var repNum = this.model.get('intention').get('evolvingFunction').get('repCount');
```

Comment:
Could you just inline these in the below lines instead of creating variables for them since they are only used once?